Update various BN/NAT articles (#10652)
ppy/osu-wiki: wiki/People/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/en.md

| Type | Cooldown | Usage |
| :-- | :-- | :-- |
| Reduced | 30 days | Applications with relatively few issues, or [Beatmap Nominator Test](/wiki/People/Beatmap_Nominators/Beatmap_Nominator_Test) fails |
| Standard | 60 days | Normal fails |
| Extended | 120 days | Applicants with major behaviour concerns |

### Rejoining after being removed from the Beatmap Nominators

Users who have recently left the Beatmap Nominators may have different requirements to rejoin compared to when they first joined. For users who have performed well prior to leaving, they may be able to rejoin with no requirements at all for a period of time. The NAT will inform the user if they have the opportunity to fast-track their way back into the Beatmap Nominators.

If you were a Beatmap Nominator before, the following requirements apply unless you have failed an application since last time you were a Beatmap Nominator:

| Type | Cooldown | Application process |
| :-- | :-- | :-- |
| Good | *None* | *None*[^good-terms] |
| Standard | 60 days | Normal |
| Activity | *None* | Normal[^activity-terms] |

## Notes

> [^good-terms]: After 1 year has passed since a Beatmap Nominator's good term resignation, it will treated as standard terms.

Reviewer: ```suggestion
[^good-terms]: A Beatmap Nominator who resigned on good terms will be treated with standard terms after 1 year.
```

> [^activity-terms]: Applicant will have to provide 3 mods in their applications, instead of at least 2.

Reviewer: ```suggestion
[^activity-terms]: Applicants will have to provide 3 mods in applications, instead of at least 2.
```
